notebooks/scenarios/bigquery/sync/01-setup-high-low-datasites.ipynb

```diff
@@ -1,5 +1,35 @@
 {
```

Comment:
<p>why are we doing this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/9281/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>